config/locales/conversion_project.en.yml

```diff
@@ -5,11 +5,22 @@ en:
       new:
         title: Add a new involuntary conversion project
         hint_html: <p class="govuk-body">Enter the school and trust information.</p><p class="govuk-body">This will create a conversion project for a school that has been issued a Directive academy order.</p>
+      create:
+        success:
+          Involuntary project successfully created
+          <p class="govuk-body">Add details of any contacts you have for the school, trust, local authority and solicitors.</p>
```

Comment:
Is there a better way of doing this, or is this okay for now?

I suppose we could pass a hash

```ruby
redirect_to project_path(@project), 
notice: {
  title: I18n.t("conversion_project.voluntary.create.success.title"), 
  body: I18n.t("conversion_project.voluntary.create.success.body") }


<% flash.each do |type, msg| %>
        <p class="govuk-notification-banner__heading"><%= msg[:title].html_safe %></p>
        <p class="govuk-body"><%= msg[:body].html_safe %></p>

```